pages/common/verilator.md

```diff
@@ -0,0 +1,20 @@
+# verilator
+
+> Converts Verilog and SystemVerilog hardware description language (HDL) designs into a C++ or SystemC model that after compiling can be executed.
+> More information: <https://veripool.org/guide/latest/>.
+
+- Build the C project in the current directory:
+
+`verilator --binary -j 0 -Wall {{path/to/source.v}}`
+
+- Create C++ executable:
+
+`verilator --cc --exe --build -j 0 -Wall {{path/to/source.cpp}} {{path/to/output.v}}`
```

Comment:
```suggestion
`verilator --cc --exe --build --build-jobs 0 -Wall {{path/to/source.cpp}} {{path/to/output.v}}`
```

Optional change: `-j` alias can be used: long options are more readable are memorable.

```suggestion
`verilator --cc --exe --build -j 0 -Wall {{path/to/output.v}} {{path/to/source.cpp}}`
```

Optional change: unexpected placeholder order: two placeholders can be swapped to conform this option specification:

```
           verilator --help
           verilator --version
           verilator --binary -j 0 [options] [source_files.v]... [opt_c_files.cpp/c/cc/a/o/so]
           verilator --cc [options] [source_files.v]... [opt_c_files.cpp/c/cc/a/o/so]
           verilator --sc [options] [source_files.v]... [opt_c_files.cpp/c/cc/a/o/so]
           verilator --lint-only -Wall [source_files.v]...
```


```suggestion
`verilator --cc --exe --build --build-jobs 0 -Wall {{path/to/output.v}} {{path/to/source.cpp}}`
```

Optional change: two fixes can be applied together.